social-network/src/YourArticle.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Author from "./Author";
+import Properties from "./Properties";
+
+
+class YourArticle extends Component {
+    constructor(props) {
+        super(props);
+    }
+
+    static propTypes = {
+        obj: PropTypes.object
+    }
+
```

Comment:
Works, may want to use PropTypes.shape({}) to dig into the object to define more keys.